### ISSUE TITLE
3.0 issues for Linux users

### ISSUE THREAD

**Issue by akirataguchi115** [mohnish/rearrange-tabs#54]
3.0 cleared the shortcuts for Linux users. I had to configure them again from the /shortcuts page. Can anyone else replicate this? Running Google Chrome on Debian 11.

**Comment by mohnish** [comment]
Hi @akirataguchi115, I just got the update too and it wiped away the shortcuts for 3/4 actions. Only the "move left" was intact. My guess is that this happened because I changed the default shortcuts. I didn't expect the update to override the custom ones. Oh boy! 🤦‍♂️ On another computer, it took about 2-3 mins before it started working even though the shortcuts weren't wiped out. I have no idea why that was the case. I hope this is a one-time issue and users are able to reconfigure the shortcuts.

Is the extension working fine after you reconfigured your keys?

PS: I tested the extension several times during the process and I didn't experience this during that time.

**Comment by akirataguchi115** [comment]
Thanks for the reply! Actually now that I try to move tab over the "end" of the tabs it should go to place 0 on the left but now it just does not do that. Did I manage to explain the unexpected behavior clear enough?

**Comment by mohnish** [comment]
Oh yeah, I think I got what you mean and that's the expected behavior now. I removed the go-around feature. I mentioned about it here: https://iam.mt/rearrange-tabs-3-0-0/

I introduced that feature early on (v1) when the option to move to front/back (v2) wasn't implemented. When I worked on the 3.0 rewrite, I carefully weighed in on which aspects of the extension seemed redundant, buggy, or causing perf issues, and this one fit the redundant category. I tried to clean up the extension and make it lean. I hope you're able to use the move back/front instead.
